src/components/contactForm/ContactForm.js

```diff
@@ -1,32 +1,60 @@
 import React from "react";
 
 export const ContactForm = ({
-  contacts,
   name,
   setName,
   phone,
   setPhone,
   email,
   setEmail,
-  handleSubmit
+  handleSubmit,
 }) => {
   return (
+    // Some name have hyphens, some have apostrophes. Let’s add support for these characters now.
     <form onSubmit={handleSubmit}>
       {/* name input */}
-    <input type="text" placeholder="name" value={name}
-        onChange={({target}) => {setName(target.value)}}
-        required />
+      <input
+        className="checkname"
+        type="text"
+        placeholder="name"
+        value={name}
+        pattern="[a-zA-Z]+(?:(?:\. |[' ])[a-zA-Z]+)*"
+        onChange={({ target }) => {
+          setName(target.value);
+        }}
+        autocomplete="on"

```

Comment:
Error in console - should this be autoComplete?

Reply:
Yes, It should e autoComplete.